pkg/operator/etcdcertsigner/etcdcertsignercontroller.go

```diff
@@ -331,6 +339,64 @@ func (c *EtcdCertSignerController) createSecretForNode(node *corev1.Node) error
 	return nil
 }
 
+func (c *EtcdCertSignerController) Get43CompatibilityNames(node *corev1.Node) []string {
```

Comment:
we would remove this entire method in 4.5?

Reply:
I added more documentation to explain

Reply:
but yes